server/serverAPI/controller/AuthenticationController.ts

```diff
@@ -74,21 +76,51 @@ export default class AuthenticationController extends BaseUserController {
                 username: user.username
             };
 
-            let token = this.tokenCreator.sign(identification, this.timeoutTimeInSeconds);
+
+            let accessToken = this.tokenCreator.sign(identification, this.accessTokenTimeoutInSeconds);
+            let refreshToken = this.tokenCreator.sign(identification, this.refreshTokenTimeoutInSeconds);
+
+            JWTStorage.getInstance().addJWT(
+                userCredentials.username,
+                new Token(accessToken, refreshToken)
+            );
 
             if (req.query.includeInfo === 'true') {
                 return this.sendSuccess(200, res, {
-                    accessToken: token,
+                    accessToken: accessToken,
+                    refreshToken: refreshToken,
                     userInfo: this.convertToUserResponse(user)
                 });
             } else {
-                return this.sendSuccess(200, res, { accessToken: token });
+                return this.sendSuccess(200, res, {
+                    accessToken: accessToken,
+                    refreshToken: refreshToken
+                });
             }
         } catch (e) {
             return e;
         }
     }
 
+    refreshToken = async (req: Request, res: Response) => {
```

Comment:
naming is confusing because you have a variable and a function called `refreshToken`. can you call this `refreshJWT` instead?